designsafe/static/scripts/projects/components/manage-project-type/manage-project-type.component.html

```diff
@@ -379,7 +379,7 @@ <h4 class="modal-title" style="border:none;">
                     <div style="margin-left: 18px;">
                         <div><strong>Simulation</strong> contain all the files you want to publish under a single DOI.</div>
                         <div><strong>Categories</strong> group files together based on shared purpose in an simulation.</div>
-                        <div><strong>Tree Diagrams</strong> relate categories to each other and to an simulation.</div>
+                        <div><strong>Related Data</strong> shows the relationship within categories and to an experiment.</div>
```

Comment:
The ticket specifies changes for Experimental Overview only. Check to see if wording should be changed for Simulation overview like here

Reply:
@rstijerina fixed